### PR TITLE
Fixes alignment of clinics-table headers

### LINKED ISSUE #247: Fix table headers positioning
The table header text is not correctly vertically aligned in some of the columns. 

Screen Shot 2021-07-22 at 7 59 23 PM

### PR DESCRIPTION
**Pull request by bridgetrosefitz**
### Comments

Fixed the alignment headers. The problem was that the sorting arrows were wrapping onto the next line on some of the headers but not on others. 

One small thing to note: The fix (`white-space: nowrap;`) changed the spacing of the columns slightly - this only really affects the contents of the left-hand column, where some of the clinic names now wrap onto three rows instead of two. 

Let me know if this is important to the team to address, and I could do one of the following fixes:
- Hard code a tiny bit more width onto the Clinic Name column 
- Decrease the margin on the parent container just above the `#waiting-lists-table` div

#### Before change

before-change

#### After change

after-change

### CODE REVIEW

File: components/CancellationList.vue [hunk 1]
```diff
@@ -181,8 +181,14 @@ export default {
         text: `${vue.$t("cancelList.header.prefecture")}:`,
         value: "prefecture",
       },
-      { text: `${vue.$t("cancelList.header.city")}:`, value: "city" },
-      { text: `${vue.$t("cancelList.header.ward")}:`, value: "ward" },
+      {
```
Comment (ann-kilzer) [inline components/CancellationList.vue:184]: Can we undo accidental layout changes? Make sure you have both prettier and eslint configured.

**Review by ann-kilzer — APPROVED** [review]
It looks great! Can we undo the accidental layout changes? (Unless I am mistaken and we had un-prettier-ed code)

Thanks for making this fix

File: components/CancellationList.vue (hunk 1 above)
Comment (bridgetrosefitz) [reply components/CancellationList.vue]: @ann-kilzer reverted! 😺

**Comment by bridgetrosefitz** [comment]
Woohoo looks great in prod!

**Pull request merged**